tests/accuracy/test_measure_optics.py

```diff
@@ -29,7 +29,7 @@ def _create_input(motion):
                     outputdir=BASE_PATH)
     optics_opt, rest = _optics_entrypoint(opt_dict)
     optics_opt.accelerator = manager.get_accelerator(rest)
-    lins = optics_measurement_test_files(opt_dict["model_dir"], dpps, motion)
+    lins = optics_measurement_test_files(opt_dict["model_dir"], dpps, motion, 1)
```

Comment:
use KW-args. this random `1` does not mean anything.

alternatively: beam_direction could be a string: "clockwise" or "counterclockwise", ("cw" "ccw"? "forward", "backward"?) which is then mapped to 1 and -1 in generate_linfiles ?

Reply:
I just saw that this is only test.. still `beam_direction = 1` here, otherwise ignore my comment